maintenance.md

```diff
@@ -733,6 +733,15 @@ opentelemetry-collector container consists of three repositories: opentelemetry-
 3. Update `K8S_VERSION` and `PAUSE_VERSION` in `Dockerfile`.
 4. Update `BRANCH` and `TAG` files.
 
+## pod-delete-rate-limiter
+
+![Regular Update](./regular_update.svg)
```

Comment:
pod-delete-rate-limiter depends on k8s libraries.
So we should update it in the Kubernetes Update.

A version is hard-coded on ENVTEST_K8S_VERSION in a Makefile. we need update it.